tests/IResearch/CMakeLists.txt

```diff
@@ -79,7 +79,6 @@ add_executable(
   arangodbtests-iresearch
   ${ARANGODB_TESTS_COMMON_SOURCES}
   ${ARANGODB_IRESEARCH_TESTS_SOURCES}
```

Comment:
Needs to remove tests common sources if we link arango_tests_basics? Or not? Can you check?

Reply:
We don't need it anymore.